sites/native-twin-docs/src/app/layout.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./globals.css";
+import { Navbar } from "@/components/Navbar";
+import { ThemeProvider } from "./theme-provider";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
+};
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  return (
+    <html lang="en">
+      <body
+        className={
+          inter.className + "  bg-gradient-to-r from-[#030712] to-[#1D1B20] "
+        }
+      >
+       <ThemeProvider
+       attribute="class"
+       defaultTheme="dark"
+       >
+        <Navbar></Navbar>
+        <div className="min-h-[100vh] py-[80px] text-white max-w-[1400px] w-[95%] m-auto">
```

Comment:
probably you need to setup any linter or formatter

Reply:
I encountered issues with Prettier in this project, but the team lead, @chrisarts, has been notified.